source/dlangbot/bugzilla.d

```diff
@@ -25,7 +25,7 @@ auto matchIssueRefs(string message)
     }
 
     // see https://github.com/github/github-services/blob/2e886f407696261bd5adfc99b16d36d5e7b50241/lib/services/bugzilla.rb#L155
-    enum issueRE = ctRegex!(`((close|fix|address)e?(s|d)? )?(ticket|bug|tracker item|issue)s?:? *([\d ,\+&#and]+)`, "i");
+    enum issueRE = regex(`((close|fix|address)e?(s|d)? )?(ticket|bug|tracker item|issue)s?:? *([\d ,\+&#and]+)`, "i");
```

Comment:
I am still trying to reduce this with dustmite to report a bug - funnily it only appears with the changes of this PR.